warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1174,6 +1186,11 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
         // need to fetch field to datatype map first
         timedFetchDatatypes(timers, "Fetch Required Datatypes", config.getQueryTree(), config);
 
+        if (!disableFieldReplacementRules) {
```

Comment:
perhaps a comment here for our future selves about how this must be called before expand multi normalized terms, so no one accidentally refactors this in the wrong order